lib/src/icons/Close.scss

```diff
@@ -0,0 +1,3 @@
+.nx-icon--close {
+  vertical-align: -0.125em;
```

Comment:
Why use `em` here? We almost never use `em`.

Reply:
> Why use em here? We almost never use em.

1. This style precisely matches how fontawesome aligns their icons
2. This would continue to work if the close button size was changed.